.gitignore

```diff
@@ -49,6 +49,7 @@ dist
 _build
 setup.bin
 setup.data
+setup.ml
```

Comment:
I think we should also add the `_oasis` file, as that is also generated by the `Makefile`.